backend/src/routes/api.ts

```diff
@@ -117,6 +120,69 @@ api.get("/v1/users/:id", async (req, res, next) => {
     next(err);
   }
 });
+api.get("/v1/users/", async (req, res, next) => {
```

Comment:
This is redundant code that duplicates the functionality in the `/v1/current-session` endpoint down below